packages/create-pantheon-decoupled-kit/src/templates/next-drupal-umami-addon/components/recipeGridItem.jsx.hbs

```diff
@@ -2,35 +2,53 @@ import Link from 'next/link';
 import Image from 'next/image';
 import { IMAGE_URL } from '../lib/constants';
 import { GradientPlaceholder } from './grid'
+{{#unless tailwindcss}}
```

Comment:
Given the error I was seeing when creating test projects, wondering if the conditional is backwards here.